plugins/gcp/resoto_plugin_gcp/resources/base.py

```diff
@@ -168,29 +168,28 @@ def add_node(self, node: GcpResourceType, source: Optional[Json] = None) -> Opti
         log.debug(f"{self.name}: add node {node}")
         node._cloud = self.cloud
         node._account = self.project
-        if source is not None and InternalZoneProp in source:
-            if zone := self.zone_by_name.get(source[InternalZoneProp]):
-                node._zone = zone
-                node._region = self.region_by_zone_name[source[InternalZoneProp]]
-                self.add_edge(node, node=zone, reverse=True)
+        if not (node._region or isinstance(node, GcpRegion)):
```

Comment:
- Why wouldn't we link a GcpRegion to a project? (or where does this happen)?
- If the resource already has a region, where is the edge created?